### ISSUE TITLE
plone.restapi is active without installing

### ISSUE THREAD

**Issue by pabo3000** [plone/plone.restapi#834]
Set up Plone 5.2 on Python 3.7. Spawn a new Plone site. Try to create an item in Plone via REST.
```
>>> import requests
>>> r = requests.post('http://localhost:8080/Plone/', headers={ 'Accept': 'application/json', 'Content-Type': 'application/json', }, json={ '@type': 'Document', 'title': 'My Document', }, auth=('admin', '******'))
>>> r
<Response [201]>
```
Imho it should not work because you have not installed plone.restapi in the configuration.

**Comment by tisto** [comment]
I agree this shouldn't be the case. The problem is that plone.rest is active always and hooks into plone.restapi then. We would need to somehow add a check to plone.restapi to check if it is installed.

**Comment by jensens** [comment]
Closing as outdated (6 years old). plone.restapi is now a core dependency in Plone 6 — being active without explicit install is intended behavior.
If you think this issue is still relevant please reopen it. Thank you for the original report!

**Comment by davisagli** [comment]
plone.restapi is still a core add-on that appears in the addons controlpanel. This is a valid issue.
